cmd/wksctl/kubeconfig/kubeconfig.go

```diff
@@ -121,6 +130,11 @@ func writeKubeconfig(ctx context.Context, cpath, mpath string) error {
 		configPath = clientcmd.RecommendedHomeFile
 	}
 
+	// Check for cluster-key in repo
+	if _, err := os.Stat(kubeconfigOptions.sshKeyPath); os.IsNotExist(err) {
+		kubeconfigOptions.sshKeyPath = "./setup/cluster-key"
+	}
```

Comment:
Oh man, this should be read out of `config.yaml` really too. Not sure how to do that though... hrm! Can we completely override the `kubeconfig` command inside of `wk` I wonder?